source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,93 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can bulk convert one or more queries, stored procedures, triggers, 
+and views in bulk by enabling the multi-select toggle. Copy and paste 
```

Comment:
(completeness)
This is a great start and really nice detail! Maybe round it off by adding one more sentence at the beginning defining bulk conversions and what they do?

Reply:
Nice!